tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -972,6 +973,9 @@ def _cudnn_rnn(inputs,
     params: the parameter buffer created for this model.
     is_training: whether this operation will be used in training or inference
     rnn_mode: one of ('lstm', 'gru', 'rnn_relu', 'rnn_tanh').
+    sequence_lengths: an int32 array representing the variable sequence lengths
+      in a batch. The size of the array has to equal the batch_size. If not
+      provided, the same sequence length will be assumed.
```

Comment:
`If not provided, the same sequence length will be assumed.` --> `Default to None, in which case sequences in the batch are assumed to have the same length, which is inferred from `inputs`.`